ports/zephyr/modmachine.c

```diff
@@ -61,3 +64,52 @@ MP_DEFINE_CONST_FUN_OBJ_0(machine_reset_cause_obj, machine_reset_cause);
 static void mp_machine_idle(void) {
     k_yield();
 }
+
+#ifdef CONFIG_PM_DEVICE
+static void mp_machine_lightsleep(size_t n_args, const mp_obj_t *args) {
+    // Check if no argument is provided
+    if (n_args == 0) {
+        mp_raise_ValueError(MP_ERROR_TEXT("value must be provided"));
+    }
+    mp_int_t milliseconds = mp_obj_get_int(args[0]);
+    // Get the UART device
+    const struct device *uart0 = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+    // Set UART device to low power state
+    pm_device_action_run(uart0, PM_DEVICE_ACTION_SUSPEND);
```

Comment:
Will this work on boards that use USB CDC as the console?  Maybe the code here will still be OK, but at least the variable should be renamed from `uart0` to `console`?